_posts/2020/2020-10-19-3-tipps-for-feedback-in-a-Scrum-Team

```diff
@@ -0,0 +1,81 @@
+
+## 1)	Don’t wait
+
+Give feedback as early as possible, directly after the incident happened or as early as possible. 
+This helps you and your colleague because your memory is still fresh, your colleague might ask you questions about details back and if he or she decides to do so, can adapt the behaviour directly. 
+Let me tell you there is nothing more awkward, than working in a team for six months and then finding out at the end of your probation period, that everyone had strong concerns about your work, but waited to last appointment to tell your manager.
+Kim Scott, author of “Radical Candor” puts this to the point: 
+“…but allowing problems to go unaddressed will only result in much bigger problems in the future, when you may be even less equipped to deal with them”
+
+## 2)	Don’t pass feedback on
+
+If you have feedback to give, please tell that person yourself. 
+And first and foremost only speak for yourself! If you tell your colleague: “The team thinks you are not capable to do this”, it will definitely leave him/her with an awkward feeling of : “How many team members do think I am unable to do my work? Why exactly do they feel like this and why did they not tell my myself?” 
+As a Scrum Master I have to stress that you may speak to your Scrum Master / Agile Coach about the matter, but it’s about you to address your feedback. 
+The Scrum Master cannot give background about what exactly annoyed you. 
+Three team members telling someone they did not like a behaviour is ten times more efficient that one consolidated Scrum Master feedback. 
+The same is true for feedback via your manager. 
+If you complain to the manager directly instead of telling the person, this may result in a negative view your manager will get on that person, lesser likelihood for raises and usually, it will take longer for the manager to give feedback than for you. 
+What would you prefer? Would you like your colleagues to do give you direct feedback or go through your manager instead?
+
+## 3)	Praise often
+
+This article is mainly focussed on giving negative feedback, but of course the same is true for praise. 
+Honest and specific praise costs you very little effort and has a huge effect. 
+This is especially true when you had talked about an improvement your colleague could make and when he actually changed the behaviour, instant praise is the best way to reinforce this.
+
+
+## How we do it at ePages
+
+First and foremost, ePages has a company culture, that values openness, honesty and direct feedback. The onboarding for new employees is characterized by often and early feedback from different levels (HR, peers and manager) and of course the new employee also gives feedback to us and to the onboarding process. All employees participate in a 360 degree feedback process at least once a year, in this process everyone is rated on Communication, Collaboration and Quality of work and peers and manger give comments on what possible improvements they see.
```

Comment:
"Communication, Collaboration,...": since we currently try to adapt it further to the teams' needs, maybe you could include a sentence, like: "We are currently thinking of how to improve to meet individual teams' needs" ?